app/src/test/kotlin/com/wire/android/feature/auth/registration/personal/CreatePersonalAccountViewPagerAdapterTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.wire.android.feature.auth.registration.personal
+
+import androidx.fragment.app.FragmentManager
+import com.wire.android.UnitTest
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+
+class CreatePersonalAccountViewPagerAdapterTest : UnitTest() {
+
+    private lateinit var createPersonalAccountViewPagerAdapter: CreatePersonalAccountViewPagerAdapter
+
+    @Mock
+    private lateinit var fragmentManager: FragmentManager
+
+    @Mock
+    private lateinit var titles: List<String>
+
+    @Before
+    fun setup() {
+        createPersonalAccountViewPagerAdapter = CreatePersonalAccountViewPagerAdapter(fragmentManager, titles)
+    }
+
+    @Test
+    fun `given getCount() is called, then return the size of the adapter`() {
+        `when`(titles.size).thenReturn(TEST_SIZE)
+        assertEquals(TEST_SIZE, createPersonalAccountViewPagerAdapter.count)
+    }
+
+    @Test
+    fun `given getPageTitle(position) is called, then return the title to show for the given position`() {
+        `when`(titles[0]).thenReturn(TEST_TITLE_EMAIL)
+        `when`(titles[1]).thenReturn(TEST_TITLE_PHONE)
+
+        assertEquals(TEST_TITLE_EMAIL.toUpperCase(), createPersonalAccountViewPagerAdapter.getPageTitle(0))
```

Comment:
Can we use assertJ, it has a fluent api (you can concatenate assertions) plus more readable code. It already exists in the dependencies. AFAIK, this is something we discussed in the Collective. 

This would look like something like this:

```kotlin
assertThat(TEST_TITLE_EMAIL.toUpperCase()).isEqualTo(createPersonalAccountViewPagerAdapter.getPageTitle(0)))
```
Here is the doc:
https://joel-costigliola.github.io/assertj/

Reply:
hmm, i missed that part. done!